PythonVisualizations/PriorityQueue.py

```diff
@@ -0,0 +1,438 @@
+import random
+from tkinter import *
+
+try:
+    from drawable import *
+    from VisualizationApp import *
+except ModuleNotFoundError:
+    from .drawable import *
+    from .VisualizationApp import *
+
+
+class PriorityQueue(VisualizationApp):
+    CELL_SIZE = 50
+    CELL_BORDER = 2
+    CELL_BORDER_COLOR = 'black'
+    ARRAY_X0 = 100
+    ARRAY_Y0 = 100
+
+    colors = ['red', 'green', 'blue', 'orange', 'yellow', 'cyan', 'magenta',
+              'dodgerblue', 'turquoise', 'grey', 'gold', 'pink']
+    nextColor = 0
```

Comment:
On lines 218-219, the `createCellValue()` method uses the color palette provided by the `drawable` module.  That makes the palette consistent with the other visualizations and allows for future customization across all visualizations.  Lines 19-20 should be removed to avoid confusion.